app/support/reservations/validations.rb

```diff
@@ -175,6 +180,14 @@ def longest_reservation_window(groups = [])
 
   private
 
+  def admin_or_ordered_by_staff?
+    admin? || order_detail.try(:created_by_user) && order_detail.created_by_user.operable_facilities.include?(facility)
```

Comment:
This is a obscure use case, but it's definitely possible, and I'm not quite sure this is behaving like it should in that situation.

* An instrument has a 12 hour cutoff
* An admin places a reservation on behalf of a user for 6 hours from now
* The user logs in and goes in to My Reservations and clicks on the reservation
* They can move the reservation

I'm not sure they should be able to do that.

I think that's part of why we use the `save_extended_validations` which saves as the acting user rather than checks how it was purchased.